src/pages/new-record.js

```diff
@@ -36,6 +36,16 @@ const NewRecord = () => {
   const navigate = useNavigate();
   const [newRecord, { isLoading }] = useNewRecordMutation();
 
+  Array.prototype.forEach.call(
+    document.querySelectorAll("input[type=text]"),
+    function (input) {
+      input.addEventListener("keyup", function () {
+        console.log("Key upped!");
+        input.value = input.value.toUpperCase();
+      });
+    }
+  );
+
```

Comment:
Text inputs are capitalized on typing. Well done!

However, it only works when the page is first visited. Creating a second record does not have the intented result.

Perhaps the useEffect hook could fix that